pom.xml

```diff
@@ -119,6 +119,12 @@
       <version>3.8.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
```

Comment:
I don't think this is a good idea. This pulls in a large lib for little gain.

Reply:
I was more interested on the `Builder`, looks more DRY and is configurable as simple annotations. 
But I agree that is a little gain for now. I'll remove the lib.